docs/Tutorials/Quorum-Dev-Quickstart.md

```diff
@@ -20,6 +20,9 @@ description: Helps to generate local GoQuorum and Ethereum blockchain networks.
 
 ## Usage
 
+You can follow this [walkthrough](https://consensys.net/quorum/products/guides/getting-started-with-consensys-quorum/) 
```

Comment:
As with my comment on the [Besu-docs PR](https://github.com/hyperledger/besu-docs/pull/725#discussion_r644265688), change "walkthrough" to "guide" or "walk-through", and remove the trailing space at the end of this line.

Reply:
done